tests/population/test_add_new_birth_cohort.py

```diff
@@ -65,6 +64,50 @@ def test_FertilityCrudeBirthRate(base_config, base_plugins):
     # TODO: Write a more rigorous test.
     assert len(pop.age) > start_population_size, 'expect new simulants'
 
+    base_config.update({
```

Comment:
We want to try not to add more integration tests like this if we can avoid it.  Maybe have ``add_new_birth_cohort`` method pass the population table into the ``_get_birth_rate`` function and make that function static.  Then you can write a proper unit test that doesn't involve setting up and running a simulation.